spec/models/survey_spec.rb

```diff
@@ -1,3 +1,31 @@
 require 'rails_helper'
 
-describe Survey do end
+describe Survey do
+  describe 'formatted_date' do
+    it 'returns the formatted date' do
+      date = DateTime.current
+      survey = create :survey, date: date
+      expect(survey.formatted_date).to eql date.strftime('%m/%d/%Y')
+    end
+  end
+
+  describe 'time' do
+    it 'returns the military time' do
+      time = DateTime.current
+      survey = create :survey, date: time
+      expect(survey.time).to eql time.strftime('%H:%M')
```

Comment:
Sat 10/29 11am-4pm (approximately): Route 35 will bypass stops on University Dr and near the Stadium due to the football game. Go UMass!